docs/release-notes/next/GH-3883-generalized-gc.md

```diff
@@ -0,0 +1,3 @@
+* Garbage collection has been rewritten, and now operates on all state trees. It is also
```

Comment:
Is this backward compatible with a database created by:
- previous version with GC enabled
- previous version without GC enabled

Reply:
Hopefully. Not tested yet :)